service/src/main/java/com/fecklessweasel/service/UniversityServlet.java

```diff
@@ -48,6 +49,7 @@ public Integer run(Connection connection)
 			// Redirect to homepage.
 			//response.sendRedirect("/");
 		request.setAttribute("longName", longName);
-		request.getRequestDispatcher("/classroom.jsp").forward(request, response);
-	}
+		request.setAttribute("state", state);
+		request.getRequestDispatcher("/UniversityPage.jsp").forward(request, response);
```

Comment:
URLs should be consistent across pages. For example, the user's profile page is in "/account/index.jsp." What this means is that the user will eventually be able to load another user's profile by going to www.fweasel.com/account?user=gundermanc.

This works because the server loads the *index.jsp* file by default in each directory.

What I propose is that you move UniversityPage.jsp to be in: */university/index.jsp* so the university url will be:
*www.fweasel.com/university?uid=35646* where uid is a unique identifier, possibly the one from the DB. What this gives us is this. The other university pages, such as department, and class can have pages here:

*/university/department/index.jsp* which will have url *www.fweasel.com/university/department?dept=EECS*
*/university/department/course.jsp* which will have url: *www.fweasel.com/university/department/course.jsp?cid=EECS454*